kratos/tests/test_element.cpp

```diff
@@ -0,0 +1,433 @@
+    {
+        //GIVES THE VECTOR WITH THE DOFS VARIABLES OF THE ELEMENT (i.e. ELEMENT DISPLACEMENTS)
+        const unsigned int dimension = GetGeometry().WorkingSpaceDimension();
+
+        if ( rValues.size() != dimension ) 
+            rValues.resize( dimension, false );
+
+        rValues[0] = GetGeometry()[0].GetSolutionStepValue( DISPLACEMENT_X, Step );
+        rValues[1] = GetGeometry()[0].GetSolutionStepValue( DISPLACEMENT_Y, Step );
+
+        if ( dimension == 3 )
+            rValues[2] = GetGeometry()[0].GetSolutionStepValue( DISPLACEMENT_Z, Step );
+    }
+
+
+    //************************************VELOCITY****************************************//
+    //************************************************************************************//
+
+    void TestElement::GetFirstDerivativesVector( Vector& rValues, int Step )
+    {
+        //GIVES THE VECTOR WITH THE TIME DERIVATIVE OF THE DOFS VARIABLES OF THE ELEMENT (i.e. ELEMENT VELOCITIES)
+        const unsigned int dimension = GetGeometry().WorkingSpaceDimension();
+
+        if ( rValues.size() != dimension ) 
+            rValues.resize( dimension, false );
+
+        rValues[0] = GetGeometry()[0].GetSolutionStepValue( VELOCITY_X, Step );
+        rValues[1] = GetGeometry()[0].GetSolutionStepValue( VELOCITY_Y, Step );
+
+        if ( dimension == 3 )
+            rValues[2] = GetGeometry()[0].GetSolutionStepValue( VELOCITY_Z, Step );
+    }
+
+    //*********************************ACCELERATION***************************************//
+    //************************************************************************************//
+
+    void TestElement::GetSecondDerivativesVector( Vector& rValues, int Step )
+    {
+        //GIVES THE VECTOR WITH THE TIME SECOND DERIVATIVE OF THE DOFS VARIABLES OF THE ELEMENT (i.e. ELEMENT ACCELERATIONS)
+        const unsigned int dimension = GetGeometry().WorkingSpaceDimension();
+
+        if ( rValues.size() != dimension ) 
+            rValues.resize( dimension, false );
+
+        rValues[0] = GetGeometry()[0].GetSolutionStepValue( ACCELERATION_X, Step );
+        rValues[1] = GetGeometry()[0].GetSolutionStepValue( ACCELERATION_Y, Step );
+
+        if ( dimension == 3 )
+            rValues[2] = GetGeometry()[0].GetSolutionStepValue( ACCELERATION_Z, Step );
+    }
+
+    //************* STARTING - ENDING  METHODS
+    //************************************************************************************//
+    //************************************************************************************//
+
+    void TestElement::Initialize()
+    {
+        KRATOS_TRY;
+
+        KRATOS_CATCH( "" );
+    }
+
+    //************************************************************************************////
+    //************************************************************************************////
+
+    void TestElement::InitializeSolutionStep( ProcessInfo& rCurrentProcessInfo )
```

Comment:
don't even define those empty methods. It will call the ones of the base class

Reply:
Ok